modules/services/cloud-bench/variables.tf

```diff
@@ -7,9 +7,39 @@ variable "account_id" {
 # optionals - with default
 #---------------------------------
 
+variable "regions" {
+  type        = list(string)
+  description = "List of regions in which to run the benchmark"
+
+  default = [
+    "af-south-1",
```

Comment:
dunno if it's what you would like, but can use tf aws regions datasource as defaults,  for a lower maintenance option (did a quick check and seems to have same region output)
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/regions

otherwise, having them alphabetically ordered would help readability :see_no_evil:

Reply:
https://github.com/sysdiglabs/terraform-aws-secure-for-cloud/pull/17/commits/71bc0d8ce2095a5e0b1cff6630ca8b29f8777dbb